_includes/homepage/hero/components/hero-infobox-desktop.html

```diff
@@ -12,13 +12,17 @@ <h1 class="hero-title mb4">
       <p class="is-hidden-touch hero-subtitle">{{- section.hero.subtitle -}}</p>
       {%- endif -%}
     </div>
-    {%- if section.hero.url and section.hero.button -%}
+    {%- if section.hero.dropdown -%}
+    <div class="is-flex" style="justify-content: center; align-content: center">
+      {% include homepage/hero/components/hero-dropdown.html %}
+    </div>
+    {% else %} {%- if section.hero.url and section.hero.button -%}
```

Comment:
Nit: should this be an `{ % elsif %}` instead

Reply:
done